__tests__/sanity.test.js

```diff
@@ -17,3 +17,11 @@ it('generates the right CSS', () => {
       expect(result.css).toBe(expected)
     })
 })
+
+it('Do not change css if tailwind is not used at all', () => {
+  return postcss([tailwind()])
+    .process('')
+    .then(result => {
+      expect(result.css).toBe('')
+    })
+})
```

Comment:
Another minor change but can you make sure this file ends with a newline character?

Reply:
done